common.py

```diff
@@ -0,0 +1,28 @@
+try:
+    from IPython.display import display, Markdown
+except:
+    pass
+
+# Helper functions
+def is_notebook():
```

Comment:
This might be useful. Usually, the graph is large, and often easier to open in Vim or some text editor, but if you think this is necessary, this should not be included here but ``brick-data``.